tests/unit/frbc/frbc_usage_forecast_element_test.py

```diff
@@ -0,0 +1,61 @@
+from datetime import timedelta, datetime, timezone as offset
+import json
+from unittest import TestCase
+import uuid
+
+from s2wsjson.common import *
+from s2wsjson.frbc import *
+
+
+class FRBCUsageForecastElementTest(TestCase):
+    def test__from_json__happy_path_full(self):
+        # Arrange
+        json_str = """
+{
```

Comment:
From this test, I noticed that the percentiles values are not being tested/generated properly:

```
lower limit <= 96 lower limit <= 68 lower limit <= 68 upper limit <= 95 upper limit <= upper limit
```

We can tackle this is a separate ticket.

Reply:
See #24